api/src/main/java/org/asynchttpclient/Request.java

```diff
@@ -112,6 +113,13 @@
     InputStream getStreamData();
 
     /**
+     * Return the current request's body as a Reactive Stream Publisher.
+     *
+     * @return a Reactive Stream Publisher representation of the current request's body.
```

Comment:
Alright, forget this, turns out I don't need it. See the upcoming commit for details.